packages/web-runtime/src/router/index.js

```diff
@@ -1,6 +1,7 @@
 import get from 'lodash-es/get.js'
 import Vue from 'vue'
-import Router from 'vue-router'
+// eslint-disable-next-line no-unused-vars
+import Router, { Route } from 'vue-router'
```

Comment:
`Route` is not used in code but in a docblock. Apparently eslint doesn't consider docblocks.